src/LocalAuthentication/LAEnums.cs

```diff
@@ -13,8 +13,7 @@ public enum LAPolicy : nint {
 		DeviceOwnerAuthentication = 2
 	}
 
-	[iOS (8,0)]
-	[Availability (Platform.Mac_10_10)]
+	[iOS (8,0), Watch (3,0), TV (10,0), Mac (10,10)]
```

Comment:
It's an enum so it's largely handled by the compiler (unlike us) and Apple often says they are available - which is more confusing than helping imo
Let's remove them since the fx are not available on the platforms